.github/workflows/daily.yml

```diff
@@ -75,5 +81,11 @@ jobs:
         script_stop: false
         script: |
           cd ${{ matrix.env }}
-          make refresh_product_tags
-          make restart_db
+          if [[ ! -e ./NO_DAILY ]]
+          then
+            make refresh_product_tags
```

Comment:
Why not refresh the products_tags ? They are recomputed from the mongo products collection, not imported from prod.